data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -235,9 +235,11 @@ def _create_tile_data(
     def _create_downloadable_data(
         self, score_county_state_merged_df: pd.DataFrame
     ) -> pd.DataFrame:
-        return score_county_state_merged_df[
-            constants.DOWNLOADABLE_SCORE_COLUMNS
-        ]
+        df = score_county_state_merged_df[constants.DOWNLOADABLE_SCORE_COLUMNS]
+
+        # rename fields
```

Comment:
remove?

Reply:
No, it's coming soon :) -- just for the GEOID10_TRACT field